src/components/LinkedInLogin.tsx

```diff
@@ -22,17 +21,6 @@ const LinkedInLogin: FunctionComponent = () => {
     clientId: process.env.REACT_APP_LINKEDIN_CLIENT_ID || "",
     redirectUri,
     scope: "r_liteprofile r_emailaddress",
-    onError: ({ error }) => {
```

Comment:
Removing this in favor of handling errors on our end through `onQueryStarted` in session api